mailarchive/mailarchive-subetha/src/java/org/sakaiproject/mailarchive/SakaiMessageHandlerFactory.java

```diff
@@ -370,30 +371,36 @@ public void data(InputStream data) throws RejectException, IOException {
                     }
                 } catch (MessagingException me) {
                     // INDICATES that the channel is NOT currently enabled so no messages can be received
-                    if (me.getMessage().startsWith(MESSAGE_ERROR_682)) {
+                	String mailSupport = StringUtils.trimToNull(serverConfigurationService.getString("mail.support"));
+                	String errMsg;
+                	if (me.getMessage().startsWith(MESSAGE_ERROR_682)) {
```

Comment:
It looks like there's a lot of redundancy in these error messages. I wonder if it can just be simplified and just do like:

```
String messageNumber = me.getMessage().replaceAll("(\\d+).+", "$1");
rb.getString("mail.support." + messagesNumber);
. . . 
```

These all look about the same. Just as an idea.